docs/rebase.md

```diff
@@ -20,13 +21,23 @@ The following describes the current rebase process in more detail.
 
 On the machine used for the rebase,
 
-* install `git`, `golang` (>= 1.17), `oc`, `yq`, and `jq`,
-* add a pull secret into `~/.pull-secret.json`, and
+* install `git`, `golang` (>= 1.18), `oc`, `yq`, and `jq`,
```

Comment:
I think we should require the rebase to be done on a development machine and reference the docs here. 
configure-vm.sh should contain the code for downloading the right `yq`

Reply:
Good idea. Done.